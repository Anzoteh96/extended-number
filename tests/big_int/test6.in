5009705101619937889
-896356105121910913
