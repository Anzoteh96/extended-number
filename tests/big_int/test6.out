Enter two numbers n and m:
n is 5009705101619937889
m is -896356105121910913
n + m is 4113348996498026976
n - m is 5906061206741848802
n * m is -4490479752697414439325206752051282657
n / m is -6
n % m is -368431529111527589
